npm-publish/bin/publish.sh

```diff
@@ -119,7 +119,13 @@ echo "✅ Released version $LOCAL_VERSION on GitHub and tagged"
 
 # Publish to NPM
 echo_title "npm publish"
-npm publish --access public
+OPTIONS="--access public"
+if [ "${PROVENANCE}" = "true" ] && [ "${CI:-}" = "true" ] && [ "${GITHUB_ACTIONS:-}" = "true" ]; then
```

Comment:
Suggest to use `==` so that all the code uses this style.

```suggestion
if [ "${PROVENANCE}" == "true" ] && [ "${CI:-}" == "true" ] && [ "${GITHUB_ACTIONS:-}" == "true" ]; then
```